src/haxelib/client/Main.hx

```diff
@@ -145,6 +145,7 @@ class Main {
 		apiVersion : "3.0",
 	};
 	static var IS_WINDOWS = (Sys.systemName() == "Windows");
+	static var PATH_SEPARATOR = if (IS_WINDOWS) ";" else ":";
```

Comment:
It is modelled after PERL5LIB and Java's CLASSPATH.

```:``` is a quite common symbol on Windows and cannot be used as the delimiter (for example ```C:\Program Files\Haxe\haxe.exe``` should not be split to ```C``` and ```\Program Files\Haxe\haxe.exe```).

Using ```;``` on Linux is arguable, ```:``` might be a sort of tradition. I do not know its history.